### PR TITLE
Fixes AB#489: Fix uncrypted password in the html

### PR DESCRIPTION
**Pull request by evgenii-nikulin**

### CODE REVIEW

File: src/Feature/Navigation/client/GlobalHeader/components/NavUser/index.tsx [hunk 1]
```diff
@@ -189,8 +187,7 @@ export default class NavUser extends Jss.SafePureComponent<NavUserProps, NavUser
         formValid: true,
       });
     }
-    const { email, password } = this.state;
 
-    StartAuth(email, password);
+    StartAuth(this.emailRef.current.value, this.passwordRef.current.value);
```
Comment (yuya-yuliya) [inline src/Feature/Navigation/client/GlobalHeader/components/NavUser/index.tsx:191]: Why you use `emailRef` if you have `email` in state?

File: src/Feature/Navigation/client/GlobalHeader/components/NavUser/index.tsx [hunk 2]
```diff
@@ -108,7 +108,7 @@ export default class NavUser extends Jss.SafePureComponent<NavUserProps, NavUser
 
   private renderSignInFrom() {
     const { authProcess, returnUrl } = this.props;
-    const { email, password, formValid } = this.state;
```
Comment (yuya-yuliya) [inline src/Feature/Navigation/client/GlobalHeader/components/NavUser/index.tsx:111]: Remove `password` from state model

File: src/Feature/Navigation/client/GlobalHeader/components/NavUser/index.tsx (hunk 1 above)
Comment (evgenii-nikulin) [reply src/Feature/Navigation/client/GlobalHeader/components/NavUser/index.tsx:191]: for similar behaviour as password

Comment (yuya-yuliya) [reply src/Feature/Navigation/client/GlobalHeader/components/NavUser/index.tsx:190]: Replace `this.emailRef.current.value` with `email` or use `emailRef` everywhere

**Pull request merged**